core/field_colour.ts

```diff
@@ -176,6 +176,11 @@ export class FieldColour extends Field<string> {
     }
   }
 
+  /**
+   * Updates text field to match the colour/style of the block.
+   *
+   * @internal
```

Comment:
If this is overridable by external devs, it can't be tagged`@internal` because you can't override `@internal` things :/ This is a semantic difference from closure `@package` tags (which this almost certainly had before).

Reply:
The function this overrides (`Field.applyColour`) is @internal.  Is that wrong too?  There are a lot of those in that file.

Reply:
I filed a separate issue for dealing with this, since it's a widespread problem: #6895